examples/experimental/i3s-17-and-debug/tile-layer/tile-layer.js

```diff
@@ -54,16 +54,7 @@ export default class TileLayer extends Tile3DLayer {
   _makeSimpleMeshLayer(tileHeader, oldLayer) {
     const content = tileHeader.content;
     const {attributes, modelMatrix, cartographicOrigin, texture, material} = content;
-    const {
-      pickable,
-      autoHighlight,
-      tileColorMode,
-      pickFeatures,
-      colorMap,
-      selectedTileId,
-      coloredTilesMap,
-      wireframe
-    } = this.props;
+    const {pickable, autoHighlight, pickFeatures, wireframe, getSimpleMeshLayerColor} = this.props;
```

Comment:
`getSimpleMeshLayerColor` is a weird name.  I feel that the fact we are rendering a SimpleMeshLayer is an implementation detail. I would choose a name that is more semantic. `getMeshColor`, `getTileColor`, `getColor` etc.

Reply:
Yes I supposed that you asked me about that. Before naming selection I looked on naming in Tile3DLayer in deck.gl.
For example for pointCloundLayer it uses `getPointColor` accessor:
![image](https://user-images.githubusercontent.com/70207219/117430673-03fb6f00-af31-11eb-84eb-2513b37d7139.png)
I followed the same naming logic for simpleMeshLayer:
![image](https://user-images.githubusercontent.com/70207219/117431020-66546f80-af31-11eb-8864-1fef099d172b.png)

I suppose `getMesh` color is more accurate here.